pkgs/development/python-modules/crc/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "crc";
+  version = "4.2.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "Nicoretti";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-h/RVMIJX+Lyted0FHNBcKY54EiirSclkBXCpAQSATq8=";
+  };
+
+  nativeBuildInputs = [ poetry-core ];
+
+  pythonImportsCheck = [ "crc" ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pytestFlagsArray = [ "test/" "--ignore=test/bench" ];
```

Comment:
Pytest will discover the tests itself, we only have to tell it not to run the benchmark tests.

We don't run them, because whether or not something is slow, that's likely nothing we can resolve.

Reply:
I would expect that directories work with `disabledTestPaths` as well.